doc/user/flycheck-versus-flymake.rst

```diff
@@ -4,111 +4,85 @@
++---------------------------+-----------------------+-----------------------+
+|`Automatic syntax checker  |by major mode and      |user-defined [#]_      |
+|selection <Syntax checker  |custom predicates      |                       |
+|selection_>`_              |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Manual syntax checker     |yes                    |yes                    |
+|selection <Manual          |                       |                       |
+|Selection_>`_              |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|`Multiple syntax checkers  |yes                    |yes                    |
+|per buffer`_               |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|Checking remote files      |said to work, but not  |partly?                |
+|via Tramp                  |officially supported   |                       |
+|                           |[#]_                   |                       |
++---------------------------+-----------------------+-----------------------+
+|`Definition of new         |single declarative     |function definition    |
+|syntax checkers`_          |function/macro         |[#]_                   |
++---------------------------+-----------------------+-----------------------+
+|`Error levels`_            |errors, warnings,      |errors, warnings,      |
+|                           |infos, custom levels   |notes, custom levels   |
++---------------------------+-----------------------+-----------------------+
+|`Error identifiers`_       |yes                    |no                     |
++---------------------------+-----------------------+-----------------------+
+|`Error parsing helpers     |for regexp, JSON and   |none                   |
+|<Error parsing_>`_         |XML                    |                       |
++---------------------------+-----------------------+-----------------------+
+|Error highlighting in      |yes                    |yes                    |
+|buffers                    |                       |                       |
++---------------------------+-----------------------+-----------------------+
+|Fringe icons for errors    |yes                    |yes (Emacs 24.1+)      |
++---------------------------+-----------------------+-----------------------+
+|`Error message display`_   |tooltip, echo area,    |tooltip only [#]_      |
+|                           |fully customizable     |                       |
++---------------------------+-----------------------+-----------------------+
+|List of all errors         |yes                    |yes                    |
++---------------------------+-----------------------+-----------------------+
```

Comment:
It might be worth it to mention that Flycheck can filter the error list by level

Reply:
Done.